extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.configuration.js

```diff
@@ -490,6 +490,13 @@ angular.module('PaperUI.controllers.configuration', [ 'PaperUI.constants' ]).con
             }
         });
     };
+
+    $scope.hasProperties = function(properties) {
```

Comment:
Hi @aounhaider1 
As I am a JavaScript newbie just one question:
For a patch I want to check if a dictionary is empty. I searched for "dictionary is empty javascript" and read some comments. I learned that there is no such function available.

I read about this ones:
* http://stackoverflow.com/a/32108184
* http://stackoverflow.com/a/4994244
* http://stackoverflow.com/a/6072692

If I understand your function correctly, it does something similar.

For the above patch, I have not introduced that a function but set an additional boolean value (you can have a look at).

WDYT about adding such a generic function that could be used for your and mine use case?